mrmustard/math/__init__.py

```diff
@@ -34,15 +34,19 @@
 
     math.cos(x)  # torch backend
 """
-
-
 import importlib
+from julia.api import Julia
 from mrmustard import settings
 
 if importlib.util.find_spec("tensorflow"):
-    from mrmustard.math.tensorflow import TFMath
+    from mrmustard.math.tensorflow_wrapper import TFMath
```

Comment:
Why this changes?
Just curious if it's based on personal preference, or if there's some conflict with leaving the original names :)

Reply:
Something about the way that Ray spins up it's new process environment and then performs imports was conflating this module with the tensorflow package. There is probably a way of fixing this that wouldn't require this rename but I couldn't figure it out after a couple hours. Here's the stackoverflow post that seemed best to indicate what the issue is: https://stackoverflow.com/questions/64509832/attributeerror-partially-initialized-module-tensorflow-has-no-attribute-conf